browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -3932,6 +3932,30 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_REWARDS_TOS_UPDATE_LINK_TEXT" desc="Terms of service text for rewards panel">
       See <ph name="TOS_START">&lt;tos&gt;</ph>Brave Rewards Terms of Service<ph name="TOS_END">&lt;/tos&gt;</ph>
     </message>
+    <message name="IDS_ALWAYS_ON_VPN" desc="Vpn setting title for always on">
+      Always-on VPN
+    </message>
+    <message name="IDS_ALWAYS_ON_VPN_KILL_SWITCH" desc="Vpn setting title for always on">
+      Always-on VPN &amp; Kill switch
+    </message>
+    <message name="IDS_HOW_TO_TURN_ON" desc="Vpn kill swicth tutorial text">
+      How to turn on:
+    </message>
+    <message name="IDS_KILL_SWITCH_TUTORIAL_TEXT_1" desc="Vpn kill swicth tutorial text">
+      1. Open your device's settings from your app drawer or tapping the button below.
+    </message>
+    <message name="IDS_KILL_SWITCH_TUTORIAL_TEXT_2" desc="Vpn kill swicth tutorial text">
+      2. Find Brave VPN and tap on the settings icon to the right.
+    </message>
+    <message name="IDS_KILL_SWITCH_TUTORIAL_TEXT_3" desc="Vpn kill swicth tutorial text">
+      3. Toggle <ph name="ALWAYS_ON_TUTORIAL_START">&lt;always_on_tutorial&gt;</ph>Always-on VPN<ph name="ALWAYS_ON_TUTORIAL_END">&lt;/always_on_tutorial&gt;</ph> to enable it.\n4. Toggle <ph name="ALWAYS_ON_TUTORIAL_2_START">&lt;always_on_tutorial_2&gt;</ph>Block connections without VPN<ph name="ALWAYS_ON_TUTORIAL_2_END">&lt;/always_on_tutorial_2&gt;</ph> to enable the kill switch.
```

Comment:
I would do 1, 2, 3, 4 rather or together or 3 and 4 as separate strings as 1, 2.